resources/home/dnanexus/utils/query.py

```
@@ -0,0 +1,487 @@
+    # Find hostname
+    workstation_hostname = file_url.split("/")[2]
+    dx_hostname = "dl.ec1.dnanex.us"
+
+    # Replace with eu specific hostname and update protocol
+    file_url = file_url.replace(workstation_hostname, dx_hostname)
+    file_url = file_url.replace("http", "https")
+
+    return file_url
+
+
+def get_cnv_call_details(reports):
```

Ruff (0.8.0)

Missing return type annotation for public function `get_cnv_call_details`

(ANN201)